### PR TITLE
chore: update grafana dashboard json

### PR DESCRIPTION
**Pull request by axiomatic-aardvark**
Fixes Grafana dashboard JSON, now it should be transferrable and anyone should be able to import it and use the Subgraph Radio dashboard, as long as their datasources are named "Prometheus" and "GraphQL"

### CODE REVIEW

**Comment by axiomatic-aardvark** [comment]
will need to update dashboard again when https://github.com/graphops/subgraph-radio/pull/5 is merged

File: grafana.json [hunk 1]
```diff
@@ -195,86 +179,68 @@
...
-          "aliasBy": "",
-          "annotationTags": "",
-          "annotationText": "",
-          "annotationTitle": "",
-          "constant": 6.5,
-          "dataPath": "data",
           "datasource": {
             "type": "prometheus",
-            "uid": "P1809F7CD0C75ACF3"
+            "uid": "xLT3_kf4k"
```
Comment (hopeyen) [inline grafana.json:197]: could you please explain this change?

**Review by hopeyen — APPROVED** [review]

File: grafana.json (hunk 1 above)
Comment (axiomatic-aardvark) [reply grafana.json]: whoops, looks like I missed it in the cleanup, should be good now

File: grafana.json [hunk 2]
```diff
@@ -21,15 +24,12 @@
   "editable": true,
   "fiscalYearStartMonth": 0,
   "graphTooltip": 0,
-  "id": 27,
+  "id": 13,
```
Comment (hopeyen) [inline grafana.json:27]: could you explain this change pls?

**Review by hopeyen — APPROVED** [review]
Should we update the datasource names from GraphQL to `SubgraphRadioGraphQL`?

**Review by hopeyen — APPROVED** [review]

**Review by hopeyen — APPROVED** [review]

**Pull request merged**